### PR TITLE
Stop/resume client on fatal crash

### PR DESCRIPTION
**Pull request by cristipufu**
Ref: https://github.com/aptabase/aptabase-maui/pull/9/

### CODE REVIEW

**Comment by TTalpey** [comment]
I like some things, but not sure about others.

I don't think it's a good idea to add the start/stop/running methods to the public API. I don't see apps actually wanting to control those, and it makes the interaction with the crashreporter more complicated. It's true that AppCenter.Analytics had some support for this, but it was a persistent setting, more oriented toward opt-out than actual API control. I'd much rather that part stayed internal for the Aptabase client, or at least, more fleshed out before committing it to the API.

That said, it's maybe good to expose the new TrackEventAsync explicitly like you've done. It allows a bit more visibility for apps that have important events to ensure the best-effort queuing was safe. Still, AppCenter.Analytics didn't seem to need it, that was handled internally to Analytics.TrackEvent.

Bottom line, I'd prefer to keep things slim and simple and not put too much machinery around this, especially in the fatal-exception path. The SDK can only do so much before the OS tears the app down, after all.
